src/components/stepsProgress/StepsProgress.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import {View, Text} from 'react-native';
+
+export const StepsProggress = ({currentStep = 1, totalSteps}) => {
+    if (totalSteps === null || totalSteps === undefined){
```

Comment:
I think this checks for more condition: `if(!totalSteps && !Number.isInteger(totalSteps))`